ansible/group_vars/searchengine-hosts.yml

```diff
@@ -5,7 +5,7 @@ database_name: idr
 database_username: omeroreadonly
 database_user_password:  "{{ idr_secret_postgresql_password_ro | default('omero') }}"
 searchenginecache_folder: /data/searchengine/searchengine/cacheddata/
-search_engineelasticsearch_docker_image: docker.elastic.co/elasticsearch/elasticsearch:7.16.2
+search_engineelasticsearch_docker_image: docker.elastic.co/elasticsearch/elasticsearch:8.8.1
 searchengine_docker_image: openmicroscopy/omero-searchengine:0.5.2
```

Comment:
I assume this PR will require another update with a new version of the Docker image once the corresponding application PR has been reviewed, merged and released

Reply:
Yes, this is the case.